### PR TITLE
US125730/sort discussion posts.

### PR DESCRIPTION
**Pull request by mbosiljevac**
image

### CODE REVIEW

File: components/left-panel/discussions/consistent-evaluation-discussion-post-page.js [hunk 1]
```diff
@@ -221,6 +233,28 @@ export class ConsistentEvaluationDiscussionPostPage extends SkeletonMixin(RtlMix
 				<div class="d2l-skeletize d2l-consistent-evaluation-discussion-evidence-body-rating-skeleton"></div>
 			</div>`;
 	}
+	_sortDiscussionPosts() {
```
Comment (amauryperez) [inline components/left-panel/discussions/consistent-evaluation-discussion-post-page.js:236]: this looks good. it feels like adding this type of logic is when we should have unit tests?

Comment (mbosiljevac) [reply components/left-panel/discussions/consistent-evaluation-discussion-post-page.js:236]: good point! I'll look into it!

**Comment by JamieTran** [comment]
Quick question, is the sorting-method saved when iterating?

**Comment by mbosiljevac** [comment]
> Quick question, is the sorting-method maintained when iterating?

good question! the value of sorting-method looks to stay the same but the discussionPosts aren't properly sorted, I'll fix that!

**Comment by amauryperez** [comment]
> Quick question, is the sorting-method maintained when iterating?

that is a good question. we should check with Fiona to see if this is something she would like or if she prefers to reset to the default

**Comment by mbosiljevac** [comment]
> > Quick question, is the sorting-method maintained when iterating?
> 
> that is a good question. we should check with Fiona to see if this is something she would like or if she prefers to reset to the default

I'll double check with her! I remember discussing this behaviour in one of the grooming sessions but I don't fully remember what we decided on/if we decided on anything

**Comment by mbosiljevac** [comment]
> > > Quick question, is the sorting-method maintained when iterating?
> > 
> > 
> > that is a good question. we should check with Fiona to see if this is something she would like or if she prefers to reset to the default
> 
> I'll double check with her! I remember discussing this behaviour in one of the grooming sessions but I don't fully remember what we decided on/if we decided on anything

Update: Confirmed with Fiona, we do want to keep filtering and sorting settings when iterating between students!

**Review by JamieTran — APPROVED** [review]

**Review by amauryperez — APPROVED** [review]

**Pull request merged**